### PR TITLE
Feature/EUI-3881-all-cases-in-all-work

### PR DESCRIPTION
**Pull request by rayliangatos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3881
https://tools.hmcts.net/jira/browse/EUI-3882

### Change description ###
All cases in all work

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  public getLocationById(id: string): Location {
+    if (id === this.ALL_LOCATIONS.id) {
+      return this.ALL_LOCATIONS;
+    }
+    return this.locations.find(loc => loc.id === id) || this.ALL_LOCATIONS;
+  }
+
+  private emitChangedEvent(): void {
+    this.selectionChanged.emit({
+      location: this.location,
```
Comment (udaydenduluri33) [inline src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.ts:93]: this.location and this.jurisdiction not found in the class?

Comment (rayliangatos) [reply src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.ts:93]: that is on get location() and get jurisdiction().
BTW this is actually meeting the requirement but I just put there to emit the data to the case table components, the remaining work should be implemented in Kuda's ticket.

**Review by kuda1992 — APPROVED** [review]
Looks fine

File: src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/work-allocation-2/components/case-manager-filter/case-manager-filter.component.ts:93]: Ok Make sense

**Review by udaydenduluri33 — APPROVED** [review]
@rayliangatos  thanks answering my comments. LGTM

**Pull request merged**